testplan/web_ui/testing/src/Report/InteractiveReport.js

```diff
@@ -497,6 +497,7 @@ class InteractiveReport extends React.Component {
           updateFilterFunc={noop}
           updateEmptyDisplayFunc={noop}
           updateTagsDisplayFunc={noop}
+          updateTimeDisplayFunc={noop}
```

Comment:
Can we make sure that the button to toggle time display is not displayed in the interactive UI?

Ideally, I think the time display toggle would be provided as a one of the extraButtons passed from the BatchReport (see code immediately below this comment for example of extraButtons usage). That way, only buttons that are common to both batch and interactive report UIs are stored in the toolbar component itself.

Reply:
I remove it from interactive UI, place it in extraButtons.